### ISSUE TITLE
Wrong calculation in "Transactions (C$X)"? On /accounts/show

### ISSUE THREAD

**Issue by heyheyhello** [firefly-iii/firefly-iii#3970]
**Bug description**
I am running Firefly III version 5.4.3, and my problem is:

I can't figure out why this value is $11.50 higher than it should be. I guess this could be a question, if not a bug, _Is the transaction total calculated to match the rows of the transaction table?_ If it's designed not to, then sorry I'm misunderstanding

![image](https://user-images.githubusercontent.com/44614862/96533911-9cefc480-1243-11eb-9fd7-d40045d7fd1a.png)

Also here:

![image](https://user-images.githubusercontent.com/44614862/96533951-b264ee80-1243-11eb-9106-ac7e0b8f310d.png)

I know you're thinking I'm probably just missing a transaction and need to reconcile, so I put the statement from my bank into a spreadsheet and checked all the transactions. They match. I used `=SUM()` in the spreadsheet and it's 446.73.

So, I do the same thing in HTML of Firefly:

![image](https://user-images.githubusercontent.com/44614862/96533557-e8ee3980-1242-11eb-9093-a28a284d7faa.png)

**It's the same value**. I have a virtual balance of $1500. There's this magical $11.50 that's...not shown on the page?

**Is the transaction total amount supposed to be the sum of table rows?** I might be misunderstanding how Firefly works

Thanks!

**Steps to reproduce**
I'm not sure how to reproduce it, but I did https://github.com/firefly-iii/firefly-iii/issues/3599#issuecomment-712530700 right before I noticed this. Sorry I don't have more information.

**Extra info**

**Bonus points**

- [x] I searched and nobody reported this bug before (I found another wrong calculation but not for this case)
- [x] I have added a stack trace from my log files 
- [x] I have added a screenshot. (two!)
- [ ] I was able to replicate it on the demo site https://demo.firefly-iii.org/
- [ ] I donated money (soon!)

Debug information generated at 2020-10-20 02:39:27 UTC for Firefly III version **5.4.3**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.4.3 |
| Firefly III API | 1.4.0 |
| PHP | 7.4.11 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 3d625a69-b2b5-4cc7-a19c-f26bb346f67a |
| Using docker? | true |
| Telemetry | false |
| Layout |  |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | info, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| DB version | 15 (exp. 15) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2020-09-23 00:00:00 |
| Session end | 2020-10-20 00:00:00 |
| Session first | 2016-09-23 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:81.0) Gecko/20100101 Firefox/81.0 |

**Comment by heyheyhello** [comment]
If I do "Reconcile this account" I also see the -$11.50

![image](https://user-images.githubusercontent.com/44614862/96536457-fc040800-1248-11eb-8a16-ee905e92d1e7.png)

**Comment by JC5** [comment]
I've seen this happen in the past, could be a few things. First things first. Could you browse to `/flush` to see what happens when you empty your cache?

**Comment by heyheyhello** [comment]
It redirects me to the homepage, which I think means it flushed. I don't see any changes unfortunately. I also went to Administration > Clear Cache

**Comment by heyheyhello** [comment]
Not sure if this is useful, and I'm not sure what sum of sums is, but here is a report of all data for visa (I only started in August):

![Screenshot_20201019-215553.png](https://user-images.githubusercontent.com/44614862/96542149-55beff00-1256-11eb-930d-11c36ba55e11.png)

Here is the month statement:

![Screenshot_20201019-215738.png](https://user-images.githubusercontent.com/44614862/96542157-59528600-1256-11eb-8fab-d5bfca0db05f.png)

The two values that stand out to me are the 435 and 446 red values, which yield the 11.50...

In the next few days if I have time I'll poke around the postgres database and see if I can't do some math in there to get 1053.27 by summing all visa transactions

**Comment by heyheyhello** [comment]
Thanks James, I appreciate you giving me the benefit of the doubt but the wasn't with Firefly. Firefly and the CSVs I had downloaded from my bank match. I collect and store these CSVs every month. However, as a last resort I re-downloaded them and compared, which lead me to find one transaction that was added to an old statement from August...

There is a notice:

![image](https://user-images.githubusercontent.com/44614862/96558473-91ff5900-1270-11eb-828d-40d6e9492859.png)

So maybe I'm wrong to have assumed a _bank_ would have reliable exports... I'll make an effort to re-download fresh copies from now on and not trust previous downloads.

Take care!
